R/basic.R

```diff
@@ -207,28 +212,13 @@ basic_initial <- function(info, n_particles, pars) {
   index <- info$index
   state <- numeric(info$len)
 
-  ## Always start with 10, again for compatibility
-  initial_I <- 10
-
-  ## This corresponds to the 15-19y age bracket for compatibility with
-  ## our first version, will be replaced by better seeding model, but
-  ## probably has limited impact.
-  seed_age_band <- 4L
-  index_I <- index[["I_A"]][[1]] + seed_age_band - 1L
-
-  ## ONS populations, subtracting the seed for pedantry.
   index_S <- index[["S"]]
-  initial_S <- pars$population
-  initial_S[seed_age_band] <- initial_S[seed_age_band] - initial_I
-
   index_N_tot <- index[["N_tot"]]
 
-  state[index_S] <- initial_S
-  state[index_I] <- initial_I
+  state[index_S] <- pars$population
   state[index_N_tot] <- sum(pars$population)
 
-  list(state = state,
-       step = pars$initial_step)
+  state
```

Comment:
@richfitz do we want to change this back to a list of `state` and `step` and we hardcode the initial step here?

Reply:
yes please